lib/testUrls.js

```diff
@@ -3,9 +3,22 @@ const { Cluster } = require('puppeteer-cluster');
 const axeScriptPath = require.resolve('axe-core');
 const { recordPage } = require('./reporter');
 
+function delay(time) {
+  return new Promise(function(resolve) { 
+      setTimeout(resolve, time)
+  });
+}
+
 async function testPage({ page, data, worker }) {
   const { url, prefix, axeOptions, crawl, urls, skipRegex, cluster, ignoreIncomplete } = data;
   await page.goto(`${prefix}${url}`, { waitUntil: 'domcontentloaded' });
+  console.log('before waiting');
+  // wait for page to be fully loaded, don't actually use this number, find a better way to determine when that is
+  await delay(8000);
```

Comment:
wait for openshift page to fully load, then the a11y errors will be different...

Reply:
Thanks for adding this, we had it accounted for [previously](https://github.com/seanforyou23/patternfly-next/blob/mobile-a11y-checks/test-a11y/index.js#L62-L65) using a selenium util. 8 seconds is fairly long.. is there a similar util for puppeteer to wait for an element to become available?

Reply:
oh, this is in no means meant to be a solution, just pointing to the issue that tests may run too early before page is loaded :)